src/site/content/en/learn/pwa/service-workers/index.md

```diff
@@ -1,11 +1,137 @@
 ---
 title: Service workers
 description: >
-  A Service Worker is a crucial component of every PWA. In this chapter, you will learn what it is, the power behind it and how to register one in your code.
+  Service workers are a fundamental part of a PWA and they allow capabilities such as fast loading regardless of the network, offline access, push notifications, and more.
 authors:
   - firt
-date: 2021-11-03
-placeholder: true
+date: 2021-12-03
 ---
 
-Coming soon!
+Users expect apps to start on slow or flaky network connections, or even when offline.
+They expect the content they've most recently interacted with, such as media tracks or tickets and itineraries, to be available and usable.
+When a request isn't possible, they expect to be told there's trouble instead of silently failing or crashing. And users wish to do it all quickly, as we can see in this study [Milliseconds make millions](/milliseconds-make-millions/), even a 0.1 second improvement in load times can improve conversion by up to 10%. In summary: users expect PWAs to be reliable and that's why we have service workers.
+
+## Hello service workers
+
+{% Img src="image/RK2djpBgopg9kzCyJbUSjhEGmnw1/iKWO7c2WNobLt30VZx9C.png",
+alt="A service worker as a middleware proxy, running device-side, between your PWA and servers, including your server and cross-domain servers.", width="800", height="354" %}
+
+When a request covered by the service worker's scope is made, including when a user is offline, the service worker intercepts the request, acting as a network proxy. It can then decide if it should serve the resource from the cache via the Cache Storage API, from the network as normally would happen without a service worker, or synthesize it from a local algorithm. Because of this, you can provide a similar experience to platform apps, including working entirely offline!
```

Comment:
```suggestion
When a request covered by the service worker's scope is made, including when a user is offline, the service worker intercepts the request, acting as a network proxy. It can then decide if it should serve the resource from the cache via the Cache Storage API, from the network as normally would happen without a service worker, or create it from a local algorithm. Because of this, you can provide a similar experience to platform apps, including working entirely offline!
```